docs/pages/04-common-properties.md

```diff
@@ -122,3 +122,19 @@ Pass the `<CommonProperties>` object into the `rightFlyoutContent` prop of Commo
         showRightFlyout={showRightFlyout}
     />
 ```
+
+
+If the `CommonProperties` component has custom width constraints, and it is nested inside single or multiple layers of `<div>` elements, special consideration is needed for proper layout behavior.
```

Comment:
Is this true `If the `CommonProperties` component has custom width constraints`?